external_services/lti.py

```diff
@@ -21,7 +25,29 @@ def __init__(self, service, user, instance, host, title, context_id=None, link_i
         link_id = link_id or "aplus{:d}".format(service.pk)
         title = title or link_id
 
-        student_id = self.external_student_id(user)
+        # Gather user information
+        if service.access_settings != LTI_ACCESS.ANON_API_NO : #service.allow_real_user_data:
```

Comment:
`if service.allow_real_user_data:` works if there is e.g. property in the model (check comment below).